R/layers2traces.R

```diff
@@ -252,7 +252,7 @@ to_basic.GeomLine <- function(data, prestats_data, layout, params, p, ...) {
 
 #' @export
 to_basic.GeomStep <- function(data, prestats_data, layout, params, p, ...) {
-  prefix_class(data, "GeomPath")
+  prefix_class(if((params$direction %||% "vh")  %in% c("vh", "hv", "mid")) ggplot2:::stairstep(data, direction = params$direction) else data, c("GeomPath", "GeomStep"))
```

Comment:
Using `:::` isn't allowed by `R CMD check`. Use `ggfun()` instead

Reply:
For some reason this breaks default `geom_step()` usage. See, for example, the change in `tests/testthat/_snaps/ggplot-step/step-gg-hv.svg`

Reply:
I think I fixed the test issues. basically just re rendering the scg solved it

Reply:
No, the tests are still broken. I don't think this is the right fix.